packages/workspace/src/tasks-runner/run-one-reporter.ts

```diff
@@ -1,7 +1,11 @@
-import { output } from '../utilities/output';
+import { output, TaskCacheStatus } from '../utilities/output';
```

Comment:
Seems unexpected that some imports were added to this file when the only change was the deprecation comment?